app/views/layouts/panoptic/application.html.erb

```diff
@@ -9,10 +9,10 @@
   </head>
   <body class="pb-4">
 
-    <%= render "shared/navbar" %>
+    <%= render "shared/panoptic/navbar" %>
```

Comment:
Since those views will only be shared within the engine, would it make sense to have the shared folder nested in `panoptic/` instead? 

Like `panoptic/shared/navbar` ?

Reply:
I honestly don't know but putting everything under `panoptic/` sounds better, I'll submit another PR